src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+/*Reset default margins*/
+html,
+body,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-active-color: #00acdc;
+  --link-color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: auto;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  background-color: #fff;
+  margin-right: -20px;
+  list-style-type: none;
+  height: 100%;
+  flex-wrap: wrap;
+}
+
+.nav__link {
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  text-decoration: none;
+  color: var(--link-color);
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  height: 100%;
+  text-transform: uppercase;
+  margin-right: 20px;
+  justify-content: center;
+  line-height: normal;
+}
```

Comment:
let's organise your styles a bit:
```css
.nav__link {
  display: flex;
  flex-direction: column;
  justify-content: center;
  height: 100%;
  margin-right: 20px;

  position: relative;

  cursor: pointer;
  text-decoration: none;
  text-transform: uppercase;

  color: var(--link-color);
  font-family: "Roboto", Arial, sans-serif;
  font-size: 12px;
  font-style: normal;
  font-weight: 500;
  line-height: normal;
}
```
easier to read, isn't it?

Reply:
✅ Done